docs/content/gds/admin/configuration/gds.en.md

```diff
@@ -1,7 +1,159 @@
+
+Note that the enabled flag is only respected if `GDS_MAINTENANCE` is `false`, otherwise maintenance mode supersedes service enabled flags. If `GDS_MEMBERS_INSECURE` is `false`, then `GDS_MEMBERS_CERTS` and `GDS_MEMBERS_CERT_POOL` are required.
+
+### Database
+
+The GDS makes use of a globally replicated key-value store for persistence. By default it uses the TrtlDB for this, but for testing or smaller deployments it can use a local LevelDB database instead. The database store is configured as follows:
+
+| EnvVar                       | Type   | Default | Description                                                                          |
+|------------------------------|--------|---------|--------------------------------------------------------------------------------------|
+| GDS_DATABASE_URL             | string |         | Required, the DSN to connect to the database on (see below for details)              |
+| GDS_DATABASE_REINDEX_ON_BOOT | bool   | false   | When the server starts, instead of loading indexes from disk, recreate and save them |
+| GDS_DATABASE_INSECURE        | bool   | false   | If set do not connect to the TrtlDB with mTLS authentication                         |
+| GDS_DATABASE_CERT_PATH       | string |         | The path to the mTLS client-side certs for database auth                             |
+| GDS_DATABASE_POOL_PATH       | string |         | The path to the mTLS public cert pools to accept server connections                  |
+
+The `GDS_DATABASE_URL` is a standard DSN with a scheme, host, path, and query parameters. In the case of the GDS the scheme can be either `trtl://` or `leveldb://`.
+
+When connecting to a TrtlDB, the host and port need to be specified with a trailing slash, e.g. `trtl://localhost:4436/`. Connecting via mTLS is only relevant when connecting to a TrtlDB. If `GDS_DATABASE_INSECURE` is `false`, then the `GDS_DATABASE_CERT_PATH` and `GDS_DATABASE_POOL_PATH` are required.
+
+When connecting to a LevelDB, the path to the directory on disk where the leveldb should be stored must be passed to the DSN. To specify a relative path, use three slashes: `leveldb:///relpath/to/db`, to specify an absolute path use four: `leveldb:////abspath/to/db`.
+
+### Email and SendGrid
+
+The GDS uses [SendGrid](https://sendgrid.com/) to send email notifications and to enable communication workflows with users and admins. Configure GDS to use SendGrid as follows:
+
+| EnvVar                 | Type   | Default                                | Description                                                                                 |
+|------------------------|--------|----------------------------------------|---------------------------------------------------------------------------------------------|
+| GDS_SERVICE_EMAIL      | string | TRISA Directory Service                | The email address used as the sender for all emails from the GDS system.                    |
```

Comment:
Some of the description sentences end in a `.` while others don't. This is a consistency nitpick and may be ignored.